dvc/remote/gdrive/__init__.py

```diff
@@ -0,0 +1,243 @@
+from __future__ import unicode_literals
+
+import os
+import posixpath
+import logging
+
+from funcy import cached_property
+from backoff import on_exception, expo
+
+from dvc.scheme import Schemes
+from dvc.path_info import CloudURLInfo
+from dvc.remote.base import RemoteBASE
+from dvc.config import Config
+from dvc.exceptions import DvcException
+from dvc.remote.gdrive.pydrive import (
+    RequestListFile,
+    RequestListFilePaginated,
+    RequestCreateFolder,
+    RequestUploadFile,
+    RequestDownloadFile,
+)
+from dvc.remote.gdrive.utils import FOLDER_MIME_TYPE
+
+logger = logging.getLogger(__name__)
+
+
+class GDriveURLInfo(CloudURLInfo):
+    @property
+    def netloc(self):
+        return self.parsed.netloc
```

Comment:
Going to remove it, thanks

> Also, do GDrive call it netloc or bucket?

I'm not sure if GDrive has analog. And seems what is more important the format of URLs DVC supports currently for GDrive: it can be gdrive://ID_OF_ANY_FOLDER (DVC files can be placed inside any remote directory by using its id) or gdrive://root/CUSTOM_PATH (`root` is reserved by GDrive; CUSTOM_PATH will be created by DVC if doesn't exist).

Supposing to use `CloudURLInfo.bucket` for now.

Reply:
@MaxRis @Suor can be resolved?

Reply:
Definition of `class GDriveURLInfo` was removed and `CloudURLInfo.bucket` is used.
Resolving this conversation.